lib/active_admin/helpers/settings.rb

```diff
@@ -22,6 +22,10 @@ def self.included(base)
     end
 
     def read_default_setting(name)
+      if (value = default_settings[name]).respond_to?(:call)
```

Comment:
@varyonic sounds good to me. I'm indifferent to the change here but thought this was a fix we could get in so its why I pushed it. I do think the app settings is over engineered. No need for all it offers. I'll leave it up to you then. Thanks!